multitenancy-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenantPersistentStreamMessageSourceTest.java

```diff
@@ -0,0 +1,145 @@
+package org.axonframework.extensions.multitenancy.autoconfig;
+
+import io.axoniq.axonserver.connector.event.PersistentStreamProperties;
+import org.axonframework.axonserver.connector.AxonServerConfiguration;
+import org.axonframework.axonserver.connector.event.axon.PersistentStreamMessageSource;
+import org.axonframework.config.Configuration;
+import org.axonframework.extensions.multitenancy.autoconfig.MultiTenantPersistentStreamMessageSource;
+import org.axonframework.extensions.multitenancy.autoconfig.TenantPersistentStreamMessageSourceFactory;
+import org.axonframework.extensions.multitenancy.components.TenantDescriptor;
+import org.axonframework.common.Registration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class MultiTenantPersistentStreamMessageSourceTest {
+
+    @Mock
+    private Configuration configuration;
+    @Mock
+    private TenantPersistentStreamMessageSourceFactory factory;
+    @Mock
+    private PersistentStreamProperties persistentStreamProperties;
+    @Mock
+    private ScheduledExecutorService scheduler;
+    @Mock
+    private PersistentStreamMessageSource mockTenantSource;
+
+    private MultiTenantPersistentStreamMessageSource source;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+        source = new MultiTenantPersistentStreamMessageSource(
+                "testName",
+                persistentStreamProperties,
+                scheduler,
+                100,
+                "testContext",
+                configuration,
+                factory
+        );
+    }
+
+    @Test
+    void testRegisterTenant() {
```

Comment:
Axon Framework and the extensions have stopped prepending test methods with `test` since the move to JUnit 5. So to align with the other repositories, I think it would be nice if this test class also wouldn't add `test-` in front of every test method.